NMS-14526: Karaf commands for Pollerd/Collectd scheduler (#6105)
OpenNMS/opennms: docs/modules/operation/pages/deep-dive/service-assurance/shell/introduction.adoc

= Shell Commands

> A number of Karaf Shell commands are made available to help administer and diagnose issues related to service assurance.

Reviewer: ```suggestion
A number of Karaf Shell commands are available to help administer and diagnose issues related to service assurance.
```

To use the commands, log into the Karaf Shell on your system using:

[source]
----
ssh -p 8101 admin@localhost
----

> NOTE: The Karaf shell uses the same credential as the web interface.

Reviewer: ```suggestion
NOTE: The Karaf shell uses the same credentials as the web interface.
```

      Users must be associated with the `ADMIN` role to access the shell.

> IMPORTANT: In order to keep the session open while executing long-running tasks without any user input add `-o ServerAliveInterval=10` to your ssh command.

Reviewer: ```suggestion
IMPORTANT: To keep the session open while performing long-running tasks without any user input add `-o ServerAliveInterval=10` to your ssh command.
```
